wicket-native-websocket/wicket-native-websocket-core/src/main/java/org/apache/wicket/protocol/ws/util/tester/TestWebSocketConnection.java

```diff
@@ -62,15 +63,44 @@ public IWebSocketConnection sendMessage(String message) throws IOException
 		return this;
 	}
 
-	@Override
+    @Override
+    public Future<Void> sendAsynchronousMessage(String message)
+    {
+        checkOpenness();
+        onOutMessage(message);
+        return null;
```

Comment:
Just delegate to `sendAsynchronousMessage(String message, long timeOut)` with a dummy value for `timeOut`